README.md

```diff
@@ -74,37 +74,41 @@ if __name__ == '__main__':
     main()
 ```
 
-## for developer
+## For developer
```

Comment:
Some level 2 section titles start with small letters others start with capital letters in this README.